alphagov/paas-cf
# [#155623416] Restructure/rename instance_groups to match cf-deployment - remove route emitter (PR 4 of 4)

[#155623416 Restructure/rename jobs to match cf-deployment](https://www.pivotaltracker.com/story/show/155623416)

What
----

We want to refactor our manifest to be based on the upstream [cf-deployment](https://github.com/cloudfoundry/cf-deployment).

In this PR, we remove the `route_emitter` instance group after moving the service to the diego cells in local mode in https://github.com/alphagov/paas-cf/pull/1267

We also rename cell to diego-cell in datadog monitors.

How to review
--------

 - Code review
 - Deploy the master branch
 - Deploy this branch

There should not be any downtime in the application availability test. We spec one or two errors on the API deployment, but not a lot.

Who can review
-----

Anyone but @keymon or @henrytk
